src/index.html

```diff
@@ -16,17 +19,32 @@
       href="style.css"
     />
   </head>
-  <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+  <body class="page__body">
+    <form
+      class="page__form"
+      action="#"
+      data-qa="big"
+    >
+      <input
+        class="search-bar search-bar--big"
+        type="text"
+        data-qa="keypress"
+        placeholder="Try &#34;Los Angeles&#34;"
```

Comment:
The class name `search-bar--big` suggests that it is a modifier for `search-bar`, but `search-bar` itself seems to be a class for styling rather than representing the meaning of the content. Consider using a class name that represents the function or content of the element instead of its style.